packages/rest-hooks/src/react-integration/provider/index.ts

```diff
@@ -17,8 +17,16 @@ CacheProvider.defaultProps = {
   ],
 };
 /* istanbul ignore next */
-if (process.env.NODE_ENV !== 'production')
+if (process.env.NODE_ENV !== 'production') {
   CacheProvider.defaultProps.managers.unshift(new DevToolsManager());
+  if (CacheProvider.defaultProps.managers.length > 1) {
+    // swap so devtools is right after networkmanager
+    [
+      CacheProvider.defaultProps.managers[1],
+      CacheProvider.defaultProps.managers[0],
+    ] = CacheProvider.defaultProps.managers.slice(0, 2);
```

Comment:
```suggestion
    ([
      CacheProvider.defaultProps.managers[1],
      CacheProvider.defaultProps.managers[0],
    ] = CacheProvider.defaultProps.managers.slice(0, 2));
```
are the parens only required for setting variables, not object properties? this is a super niche/rare way to use destructuring, so i'm not certain :-)

Reply:
sans parens is what prettier output. i think object properties is needed for the parse to distinguish from scope blocks. but with [] this is not necessary.